simcase-game/src/views/Welcome.js

```diff
@@ -17,12 +17,12 @@ const container = {
 function Welcome() {
   React.useState(() => {
     //Line to add data to our firebase data
-    fire
-      .firestore()
-      .collection("data")
-      .add({
-        prueba: "test"
-      });
+    // fire
```

Comment:
This commented lines of code would be necessary in this view?

Reply:
Hmm I do not think so, going to delete this.